service/src/main/java/uk/gov/hmcts/reform/fpl/service/summary/CaseSummaryPeopleInCaseGenerator.java

```diff
@@ -51,21 +54,42 @@ private String generateSummaryCafcassGuardian(CaseData caseData) {
     }
 
     private String generateSummaryMainContactEmail(CaseData caseData) {
-
         if (isNotEmpty(caseData.getLocalAuthorities())) {
-            return caseData.getDesignatedLocalAuthority().getMainContact()
-                .map(Colleague::getEmail)
-                .orElse(null);
+            if (nonNull(caseData.getDesignatedLocalAuthority())) {
+                return caseData.getDesignatedLocalAuthority().getMainContact()
+                    .map(Colleague::getEmail)
+                    .orElse(null);
+            } else {
+                LocalAuthority applicant = caseData.getLocalAuthorities().stream()
+                    .map(Element::getValue)
+                    .findFirst()
+                    .orElse(null);
+
+                return nonNull(applicant) ? applicant.getMainContact()
+                    .map(Colleague::getEmail)
+                    .orElse(null) : null;
```

Comment:
Same as above wrt. null applicants

Reply:
Same as above, getLocalAuthorities simply returns all applicants and it shouldn't actually ever be null to be fair as there HAS to be an applicant unless something has gone horribly wrong